reliability-engineering/terraform/modules/concourse-team-iam-credentials/cred-rotation.tf

```diff
@@ -53,7 +55,9 @@ resource "aws_lambda_permission" "allow_cloudwatch_to_call_sts_creds_to_ssm" {
   action        = "lambda:InvokeFunction"
   function_name = aws_lambda_function.sts_creds_to_ssm.function_name
   principal     = "events.amazonaws.com"
-  source_arn    = aws_cloudwatch_event_rule.every_ten_minutes.arn
+  source_arn    = aws_cloudwatch_event_rule.every_ten_minutes[each.key].arn
```

Comment:
not 100% sure ... but do you need to remove the `statement_id` now that this is a "many resources" thing? ... or make the `statement_id` uniq somehow